e2etests/todo_test.go

```diff
@@ -14,19 +14,22 @@ func testTodo(t *testing.T) {
 container.first -> container.second: 1->2
 container -> container.second: c->2
 `,
+			dagreFeatureError: `Connection "(container -> container.second)[0]" goes from a container to a descendant, but layout engine "dagre" does not support this.`,
```

Comment:
Can these be moved into `stable` then?

Reply:
I think we still would like to fix this for dagre if we can find a solution